### DISCUSSION TITLE
how to switch to `pnpm`

### DISCUSSION THREAD

**Discussion by zilahir** [resend/react-email#866] [Q&A]
I've recently switched to `pnpm workspace`, but it seems like running the emails in dev mode locally will always re-create the `yarn.lock` file, and installing the dependencies locally. 

Is there a way to manually switch to use `pnpm` ?

**Comment by osdiab** [comment]
i really would prefer if this library just generated routes for a NextJS app or something at a path you specify, and that you can independently configure it however you'd like; rather than it making decisions that only work for users who happen to use the specific architecture that the resend devs prefer. Initial impressions of the dev experience here have been very lackluster.

**Reply by osdiab** [reply]
but for now I'm just making a separate dev server myself and not using the `react-email` one since it's way too opinionated.
